### PR TITLE
CTI-STIX-UI Checkin

### PR DESCRIPTION
**Pull request by infosec-alchemist**
Checkin includes changes from #20 .  Also adds SSL.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,25 +1,78 @@
...
+    volumes:
+     - ./certs/:/etc/pki/tls/certs
+    entrypoint:
+     - ember
+     - server
+     - -live-reload-port
+     - "50000"
+     - -ssl
+     - -ssl-key
+     - /etc/pki/tls/certs/localhost.key
```
Comment (exceptionfactory) [inline docker-compose.yml]: This should be changed from localhost.key to server.key to match the generated file.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,25 +1,78 @@
...
+     - -ssl-cert
+     - /etc/pki/tls/certs/localhost.crt
```
Comment (exceptionfactory) [inline docker-compose.yml]: This should be changed from localhost.crt to server.crt to match the generated file.

File: cti-stix-ui/app/routes/relationship-new.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export default Ember.Route.extend({
+    /**
+     * Model sets initial model for creating new records
+     * 
+     * @return {Object} Promise Object
+     */
+    model() {
+        const hash = {};
+        hash.help = {
+            description: "Relationships are what makes the Unfetter Discover project unique.   One type of relationship is created "+
```
Comment (exceptionfactory) [inline cti-stix-ui/app/routes/relationship-new.js]: Change or remove reference to Unfetter Discover.

File: cti-stix-ui/app/routes/relationship.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    /**
+     * Model calls findRecord for individual record and for source and target referenced objects
+     * 
+     * @param {Object} parameters Parameters Object
+     * @return {Object} Promise Object
+     */
+    model(parameters) {
+        const hash = {};
+        hash.help = {
+            description: "Relationships are what makes the Unfetter Discover project unique.   One type of relationship is created to "+
```
Comment (exceptionfactory) [inline cti-stix-ui/app/routes/relationship.js]: Change or remove reference to Unfetter Discover.

File: cti-stix-ui/app/routes/relationships.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+export default Ember.Route.extend({
+    /**
+     * Model queries for collection of records
+     * 
+     * @return {Object} Promise Object
+     */
+    model() {
+        const hash = {};
+        hash.help = {
+            description: "Relationships are what makes the Unfetter Discover project unique.   One type of relationship is created to "+
```
Comment (exceptionfactory) [inline cti-stix-ui/app/routes/relationships.js]: Change or remove reference to Unfetter Discover.

File: cti-stix-ui/package.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "cti-stix-ui",
+  "version": "0.1.1",
+  "description": "Unfetter Discover UI",
```
Comment (exceptionfactory) [inline cti-stix-ui/package.json]: Change description to CTI STIX UI.

File: docker-compose.yml (hunk 1 above)
Comment (infosec-alchemist) [inline docker-compose.yml]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (infosec-alchemist) [inline docker-compose.yml]: fixed. Renamed to server.*

**Pull request merged**